package.json

```diff
@@ -32,6 +32,7 @@
     "core-js": "^2.4.1",
     "crypto": "0.0.3",
     "debug": "~2.6.3",
+    "email-verify": "^0.2.1",
```

Comment:
Is the email-verify module functioning as expected ?

Reply:
I think it's not. It was giving status 400 on my college mail ID.